README.md

```diff
@@ -860,6 +860,9 @@ imgGroup = L.distortableCollection({
 ```
 
 ---
+## MISCELLENEOUS
```

Comment:
Hi @Jennifer-tech thank you for this PR! I believe this file is a separate issue. Could you reset it so it's not included in the PR? You could try this -- https://stackoverflow.com/questions/7147270/hard-reset-of-a-single-file or just copy the version from the publiclab main branch (https://github.com/publiclab/Leaflet.DistortableImage) over your own and add it so there are no changes shown here. Let me know if you need help!

Reply:
Going through it now, thank you for the feedback @jywarren

Reply:
Hi @jywarren, I have resolved the merge conflict. Please you can now review it